pic32/cores/pic32/WInterrupts.c

```diff
@@ -263,12 +265,22 @@ void __attribute__((nomips16,at_vector(_EXTERNAL_0_VECTOR),interrupt(IPL4SRS)))
 void __attribute__((interrupt(),nomips16)) ExtInt0Handler(void)
 #endif
 {
-
+	VEC_ENT_CODE(EXT_INT0,INT0IF,PIN_INT0);
```

Comment:
I can see nothing in the MAX3421E that requires level interrupts on the MCU. The "level" referred to in the datasheet is the behaviour of the INT output pin, not the required interrupt type of the MCU. Yes, you can be lazy and use level interrupts on the MCU to repeatedly poll the INT pin, but personally that would be furthest method from my mind.  With the INT pin set to level you don't even need to use an interrupt pin - you can just poll it. You only need to use an interrupt pin when you have the INT set to edge mode.